students/nartowska_karina/lesson_12_debugging/verbose_output.py

```diff
@@ -0,0 +1,55 @@
+import re
+import glob
+import argparse
+import sys
+import logging
+
+
+LOGGER_LEVELS = {"debug": logging.DEBUG,
+                 "info": logging.INFO,
+                 "warning": logging.WARNING,
+                 "error": logging.ERROR}
+
+
+def check_arg(args=None):
+    parser = argparse.ArgumentParser(description='Regex search')
+    parser.add_argument('-r', '--regex',
+                        help='search regex',
+                        required=True)
+
+    parser.add_argument('-v', '--verbose',
+                        help='verbose output: debug, info, warning, error',
+                        default='debug')
+
+    results = parser.parse_args(args)
+    return (results.regex,
+            results.verbose)
+
+
+regex, verbose = check_arg(sys.argv)
+logging.basicConfig(level=LOGGER_LEVELS[verbose],
+                    format='%(asctime)s - %(levelname)s - %(message)s')
```

Comment:
There is no need to call twice ```check_arg(args=None):``` function in your program. You can move lines 29 - 31 to 48 line, in a result your ```main()``` function will be looks following:
```
def main():
    regex, verbose = check_arg(sys.argv[1:])
    logging.basicConfig(level=LOGGER_LEVELS[verbose],
                        format='%(asctime)s - %(levelname)s - %(message)s')
    logging.debug("Searching regex in .txt files: " + regex)
    regex_search = re.compile(regex)
    search_regex(regex_search)
```